iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.cpp

```diff
@@ -321,10 +321,11 @@ struct NodeSymbolsProvider : IECSymbolProvider
         if (extendedData.HasECClassId())
             classIds.insert(extendedData.GetECClassId());
         else if (ctx.m_node->GetKey()->AsECInstanceNodeKey())
-            {
             for (auto const& key : ctx.m_node->GetKey()->AsECInstanceNodeKey()->GetInstanceKeys())
                 classIds.insert(key.GetClass()->GetId());
-            }
+        else if (ctx.m_node->GetKey()->AsLabelGroupingNodeKey() && ctx.m_node->GetKey()->AsLabelGroupingNodeKey()->GetGroupedInstanceKeys() != nullptr)
+            for(auto const& key : *ctx.m_node->GetKey()->AsLabelGroupingNodeKey()->GetGroupedInstanceKeys())
+                classIds.insert(key.GetClassId());

```

Comment:
Formatting:
- if a block has more than 1 line (no matter if it's one or more statements), it should be wrapped with curly braces
- missing space after `for`